ui/components/src/TimeSeriesTooltip/focused-series.ts

```diff
@@ -15,6 +15,11 @@ import { ECharts as EChartsInstance } from 'echarts/core';
 import { formatValue, UnitOptions, EChartsDataFormat } from '../model';
 import { CursorData, TOOLTIP_DATE_FORMAT, TOOLTIP_MAX_ITEMS } from './tooltip-model';
 
+// increase multipliers to show more series in tooltip
+export const DEFAULT_NEARBY_SERIES_RANGE_MULTIPLIER = 5.5; // adjusts how many focused series show in tooltip
```

Comment:
I ended up renaming these constants so they are hopefully more clear, "narrow" was a little confusing and could make people think the range controls the width of the nearby series highlighted area, "shorten" is more accurate (since it adjusts yBuffer variable) but sounds weird so I landed on INCREASE_FOCUSED_SERIES_MULTIPLIER and REDUCE_FOCUSED_SERIES_MULTIPLIER for now